main_app/templates/base.html

```diff
@@ -16,37 +16,17 @@
     </head>
     <body>
         <header class="">
-            <ul id="adddropdown" class="dropdown-content">
-                <li><a href="{% url 'plant_add' %}">Add a plant</a></li>
-                {% if request.user.is_active and request.user.is_staff %}
-                <li><a href="{% url 'plant_create' %}">Create plant entry in DB</a></li>
-                {% endif %}
-            </ul>
-            <ul id="viewdropdown" class="dropdown-content">
-                <li><a href="{% url 'plants_index' %}">My plants</a></li>
-                <li><a href="{% url 'plantsDB_index' %}">Plants DB</a></li>
-            </ul>
-            <div class="parallax-container">
```

Comment:
Parallax effect removed - please remove corresponding commented code from script at the bottom unless usign parallax container somewhere else.

Reply:
Parallax effect removed including script.